gsw_oceanographic_toolbox.c

```diff
@@ -1,6 +1,6 @@
 /*
 **  $Id: gsw_oceanographic_toolbox-head,v c61271a7810d 2016/08/19 20:04:03 fdelahoyde $
```

Comment:
Minor: shouldn't the Id be removed, since it refers to an older version-control system?